script.js

```diff
@@ -1,129 +1,173 @@
-/*
- * jQuery v1.9.1 included
- */
+document.addEventListener('DOMContentLoaded', function() {
 
-$(document).ready(function() {
+  function matches (element, selector) {
+    return Element.prototype.matches && element.matches(selector)
+      || Element.prototype.msMatchesSelector && element.msMatchesSelector(selector)
+      || Element.prototype.webkitMatchesSelector && element.webkitMatchesSelector(selector);
+  }
+
+  function closest (element, selector) {
+    if (Element.prototype.closest) {
+      return element.closest(selector);
+    }
+    do {
+      if (matches(element, selector)) return element;
+      element = element.parentElement || element.parentNode;
+    } while (element !== null && element.nodeType === 1);
+    return null;
+  }
 
   // social share popups
-  $(".share a").click(function(e) {
-    e.preventDefault();
-    window.open(this.href, "", "height = 500, width = 500");
+  [].forEach.call(document.querySelectorAll('.share a'), function(anchor) {
```

Comment:
could we use `Array.prototype` instead of `[]`? Looks a bit less hacky to me
Also in line 51, 96, 166